shipment_advice_planner_toursolver/models/shipment_advice.py

```diff
@@ -14,3 +18,150 @@ class ShipmentAdvice(models.Model):
     toursolver_task_id = fields.Many2one(
         comodel_name="toursolver.task", string="Toursolver Task", readonly=True
     )
+    # Toursolver stats
+    toursolver_nbr_tours = fields.Integer(
+        string="Number of Tours", compute="_compute_toursolver_stats", store=True
```

Comment:
Isn’t it better to have these fields in `toursolver.task` model and add related fields in `shipment.advice`?